cmd/libs/go2idl/client-gen/generators/client_generator.go

```diff
@@ -120,7 +120,7 @@ func packageForGroup(gv unversioned.GroupVersion, typeList []*types.Type, packag
 			return generators
 		},
 		FilterFunc: func(c *generator.Context, t *types.Type) bool {
-			return types.ExtractCommentTags("+", t.SecondClosestCommentLines)["genclient"] == "true"
+			return types.ExtractCommentTags("+", t.SecondClosestCommentLines)["genclient"] != nil
```

Comment:
I think that all changes like this one are actually incorrect, because you can potentially set "genclient=false" in the file and this would be treated as "genclient=true" after your change.